nextcord/errors.py

```diff
@@ -281,3 +284,45 @@ class InteractionResponded(ClientException):
     def __init__(self, interaction: Interaction):
         self.interaction: Interaction = interaction
         super().__init__('This interaction has already been responded to before')
+
+class ApplicationError(DiscordException):
+    r"""The base exception type for all command related errors.
+
+    This inherits from :exc:`nextcord.DiscordException`.
```

Comment:
Isnt discordexception for exceptions caused by discord?

Reply:
DiscordException is for any errors raised by the library, as the description says:
"Base exception class for nextcord. Ideally speaking, this could be caught to handle any exceptions raised from this library."